internal/services/task.go

```diff
@@ -0,0 +1,132 @@
+package services
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/DianaTalk/go_final_project/internal/models"
+)
+
+type TaskService struct {
+	db *sql.DB
+}
+
+func NewTaskService(db *sql.DB) *TaskService {
+	return &TaskService{db: db}
+}
+
+func (s *TaskService) CreateTask(t models.Task) (int64, error) {
+	res, err := s.db.Exec(
+		"INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)",
+		t.Date, t.Title, t.Comment, t.Repeat,
+	)
+	if err != nil {
+		return 0, fmt.Errorf("ошибка создания задачи: %v", err)
```

Comment:
Большой плюс за оборачивание ошибок, но `%v` используется только для вывода, а тут нужен `%w` (wrap)
```suggestion
		return 0, fmt.Errorf("ошибка создания задачи: %w", err)
```